app/styles/tailwind.css

```diff
@@ -1,230 +1,3 @@
-/* Web fonts */
-
-/* 1. Google Fonts */
-
-@font-face {
-  font-family: 'Catamaran';
-  font-style: normal;
-  font-weight: 100;
-  src: url('/fonts/Catamaran-Thin.woff2') format('woff2');
```

Comment:
these paths cannot work on production, where:
a. `/fonts` hits the Rails app
b. the hash digest is missing in the file name.